File: AkkaCustomExtension/src/main/scala/org/akka/essentials/custom/extension/example/MyActor.scala
package org.akka.essentials.custom.extension.example
import akka.actor.Actor

class MyActor extends Actor {
  def receive: Receive = {
    case _ =>
      val mysqlSetting = MySQLJDBCSettings(context.system)
      println(mysqlSetting.DB_USER_NAME)
      println(mysqlSetting.DB_USER_PASSWORD)
  }
}

File: AkkaCustomExtension/src/main/java/org/akka/essentials/custom/extension/example/MyActor.java
package org.akka.essentials.custom.extension.example;

import akka.actor.UntypedActor;

public class MyActor extends UntypedActor {

	@Override
	public void onReceive(Object message) throws Exception {
		MySQLJDBCSettingsImpl mysqlSetting = MySQLJDBCSettings.SettingsProvider
				.get(getContext().system());

		System.out.println(mysqlSetting.DB_USER_NAME);
		System.out.println(mysqlSetting.DB_USER_PASSWORD);

	}

}
